app/views/layouts/base.html.slim

```diff
@@ -14,5 +14,5 @@ html lang="en-US"
     = yield :styles
 
   body
-    = render "application/alerts"
+    = content_for?(:alerts) ? render(inline: '') : render('application/alerts')
```

Comment:
I want this to render nothing, but was having trouble getting that working. That's why I have `render:(inline: '')`.